src/client/dfuse/dfuse_core.c

```diff
@@ -449,7 +450,9 @@ dfuse_pool_connect_by_label(struct dfuse_projection_info *fs_handle, const char
 	*_dfp = dfp;
 	return rc;
 err_disconnect:
-	daos_pool_disconnect(dfp->dfp_poh, NULL);
+	ret = daos_pool_disconnect(dfp->dfp_poh, NULL);
+	if (ret)
+		DFUSE_TRA_WARNING(dfp, "Failed to disconnect pool: "DF_RC, DP_RC(ret));
```

Comment:
This change is fine, but it's really pandering to tool as it doesn't make an practical difference to correctness.  It would be interesting to compile with logging as a no-op and see what coverity complains about then, I bet it would be a lot.

Reply:
I agree. coverity generates a lot of warnings. Probably only a small part are necessary to be addressed.

Reply:
That wasn't my point.  This change will make coverity happy but it won't do anything to the behaviour of the code.